Makefile

```diff
@@ -244,9 +243,9 @@ ifneq ("$(I_AM_USING_VERDACCIO)", "I_AM_SURE")
 	echo "You probably don't know what you are doing"
 	exit 1
 endif
-	$(YARN) lerna version $(VERSION) --exclude-dependents --force-publish=$(FORCE_PUBLISH)  --no-push --yes --tag-version-prefix="version-e2e-test-"
+	$(YARN) release-tool version $(VERSION) --all --yes --tag-version-prefix="version-e2e-test-"
```

Comment:
Note: I added `--all` to publish every package because as suggested elsewhere by @JLHwung, but the `lerna` command only published (in the tests) the packages whose source changed.

Publishing everything allows us to also catch bugs, for example, in the build process that might affect packages without actual code changes.